code/datums/actions/mobs/projectileattack.dm

```diff
@@ -16,6 +16,8 @@
 	var/default_projectile_spread = 0
 	/// The multiplier to the projectiles speed (a value of 2 makes it twice as slow, 0.5 makes it twice as fast)
 	var/projectile_speed_multiplier = 1
+	/// How long is pre-attack cooldown
+	var/pre_attack_cooldown = 10 SECONDS
```

Comment:
what does this actually do? why are you overriding it on projectile attack when its on every mob cooldown ability?